inductiva/_cli/cmd_storage/remove.py

```diff
@@ -1,20 +1,45 @@
 """Remove the user's remote storage contents via CLI."""
 
-from inductiva import storage
+import sys
+
+from inductiva import constants, storage
 
 
 def remove(args):
     """Remove user's remote storage contents."""
-    path = args.path
+    paths = args.path
     confirm = args.confirm
+    all_paths = args.all
+
+    if paths and all_paths:
+        print(
+            "inductiva storage remove: error: "
+            "argument path not allowed with argument --all",
+            file=sys.stderr)
+        sys.exit(1)
 
     if not confirm:
-        prompt = input(f"Are you sure you want to remove {path}? (y/n)")
+        if all_paths:
+            prompt = input("Are you sure you want to remove everything "
+                           "from your remote storage space (y/[N])? ")
```

Comment:
This is just being picky at the moment, but can this message be similar to the ones below and the `everything` be in CAPS LOCK for awarenes..